central/sensor/service/connection/sensorevents.go

```diff
@@ -76,6 +78,7 @@ func (s *sensorEventHandler) addMultiplexed(ctx context.Context, msg *central.Ms
 	var workerType string
 	switch event.Resource.(type) {
 	case *central.SensorEvent_Synced:
+		s.initSyncMgr.Remove(s.cluster.GetId())
```

Comment:
Move this to after reconciliation if we really want to ensure that there is low load on the system. Reconciliation is pretty expensive